src/axom/primal/geometry/NURBSPatch.hpp

```diff
@@ -1168,6 +1323,9 @@ class NURBSPatch
   /*!
    * \brief Reverses the order of one direction of the NURBS patch's control points and weights
    *
+   * This method does not change the position of the patch in space, 
+   *  including the trimming curves. It does reverse all normal vectors.
```

Comment:
(Clarity)

Something like the following might be somewhat clearer:
```suggestion
   * This method does not affect the position of the patch in space, or its trimming curves,
   * but it does reverse the patch's normal vectors.
```